include/exadg/time_integration/solver_info_data.h

```diff
@@ -54,10 +55,11 @@ struct SolverInfoData
     print_parameter(pcout, "Interval time steps", interval_time_steps);
   }
 
+  // TODO: here we might also be able to use timecontrol
```

Comment:
Principally yes, but note the comment below regarding the variable `old_time_step_number`.

Reply:
We also have `wall_time` here, so it might be ok to not change this implementation.